src/main/java/org/ecocompass/core/Main.java

```diff
@@ -18,6 +19,10 @@ public static void main(String[] args) throws Exception {
 
         Graph graph =  overpass.createGraphFromOverpassQuery(response);
 
+        KDTree tree = overpass.createTreeFromGraph(graph);
+        //double[] point = {101234, 12235};
```

Comment:
has this been tested with points from Graph object?

Reply:
Not yet but I need some points and graph points to test it. I will test it in a while.